rest-api-with-fastify/package.json

```diff
@@ -14,10 +14,10 @@
     "build": "exit 0"
   },
   "dependencies": {
-    "@romcal/calendar.france": "3.0.0-dev.67",
-    "@romcal/calendar.general-roman": "3.0.0-dev.67",
-    "fastify": "^4.12.0",
-    "romcal": "3.0.0-dev.67"
+    "@romcal/calendar.france": "3.0.0-dev.77",
+    "@romcal/calendar.general-roman": "3.0.0-dev.77",
+    "fastify": "^4.28.0",
```

Comment:
You have also update the Fastify version. It might be a good idea to say so in the PR description.